packages/flutter_driver/lib/src/driver.dart

```diff
@@ -137,7 +145,9 @@ class FlutterDriver {
       isolate = await vm.isolates.first.loadRunnable();
     }
 
-    FlutterDriver driver = new FlutterDriver.connectedTo(client, connection.peer, isolate);
+    FlutterDriver driver = new FlutterDriver.connectedTo(
+        client, connection.peer, isolate,
+        printCommunication: printCommunication);
```

Comment:
nit: if you have a newline after an open parenthesis, you should have a newline before the closing one